lib/internal/streams/readable.js

```diff
@@ -71,6 +72,7 @@ const {
 const { validateObject } = require('internal/validators');
 
 const kPaused = Symbol('kPaused');
+const originalAsyncIteratorPrototype = SymbolFor('nodejs.stream.originalAsyncIteratorPrototype');
```

Comment:
We are using `nodejs.internal` for other non-documented Symbol: https://github.com/nodejs/node/blob/007a85ce83b42284e8fb83733cfe3a60c1fe119c/lib/internal/per_context/messageport.js#L15-L17

Should we use that here as well? Or should we document it?

Reply:
Oh! I think it should remain undocumented then this should be something like `nodejs.internal.stream. originalAsyncIteratorPrototype`?

Reply:
Well that doesn't seem to be consistent, those are also internal and using `nodejs.stream` namespace: https://github.com/nodejs/node/blob/8c60add65b887208c6800540968a271d6f953545/lib/internal/streams/utils.js#L9-L17

Please disregard. Although if we had a comment like the one in `streams/utils.js`, that'd be nice.

Reply:
sure adding a comment!

Reply:
Done

Reply:
I don't think we need a `SymbolFor` here.

Reply:
So `Symbol` only and exporting that?

Reply:
Is there is diff approach you have in mind, other than symbols/exporting i dont get anything in mind @mcollina

Reply:
ping @mcollina

Reply:
Can you please avoid using a `SymbolFor` here?

Reply:
So instead of SymbolFor would exporting a symbol be acceptable?

Reply:
In this case yes (via the internal module)

Reply:
Reverted for now, I shall add it in a follow up!